heart-rate-alert/lambda.py

```diff
@@ -0,0 +1,75 @@
+import boto3
+from botocore.exceptions import ClientError
+import datetime
+
+def send_email(event):
+    SENDER = "trainee.alex.skowronski@sigmalabs.co.uk"
+    RECIPIENT = "three.musketeers.deloton@gmail.com"
+
+    AWS_REGION = "eu-west-2"
+
+    """The subject line for the email."""
+    SUBJECT = "Heart Rate exceeding normal levels"
+
+    """The email body for recipients with non-HTML email clients."""
+    BODY_TEXT = ("You should be aware that your heart rate is exceeding the normal level of {}\r\n"
```

Comment:
How will the normal level for each unique rider be added here? Does this need to be included in the event sent by the EC2?

Reply:
Later on you've added `{event["limit_heart_rate"]}` should that be here as well?